config/pipeline.yaml

```diff
@@ -244,6 +248,13 @@ jobs:
       collections: dt
       job_timeout: 10
 
+  # amd64-only temporary
+  sleep:
+    template: sleep.jinja2
+    kind: test
+    params:
+      rootfs: http://storage.kernelci.org/images/rootfs/debian/bullseye/20240129.0/amd64
```

Comment:
With the latest changes, you can now rename it to `nfsroot` (and maybe s/amd64/{arch}/ for future-proofness?)